.gitignore

```diff
@@ -171,6 +171,5 @@ packages/
 /Utest/Validation/7test_BO/jpsreport_ini.xml
 
 /doc*
-/cmake-*
```

Comment:
Removing this will add the default CLion build directories

Reply:
fixed on latest commit